### PR TITLE
 tidy(common): Split off slogger from common 

### PR DESCRIPTION
**Pull request by uristdwarf**
This is another split of the common library to help encapsulate and
reduce size/usage of the common library, for both new and future
libraries.

### CODE REVIEW

**Comment by uristdwarf** [comment]
N.B! This branch needs to be rebased once #96 is merged. It's the second part

**Comment by uristdwarf** [comment]
Putting to draft, waiting for #96

**Review by aNeutrino — APPROVED** [review]
LGTM, however it will conflict with moving sfserr from common into errors right ?

**Review by rolysr — APPROVED** [review]
LGTM

File: src/common/CMakeLists.txt [hunk 1]
```diff
@@ -18,7 +18,9 @@ endif()
 
 
 shared_add_library(sfscommon ${COMMON_SOURCES})
-shared_target_link_libraries(sfscommon ${YAML_CPP_LIBRARIES} STATIC sfserr SHARED sfserr_pic)
+shared_target_link_libraries(
```
Comment (ralcolea) [inline src/common/CMakeLists.txt:21]: I think is more clear and easier to identify link errors if you separate the link to sfserr and slogger in different lines, i.e keep the original line and add a new one to link againt `slogger` library.

**Review by ralcolea — APPROVED** [review]
LGTM. I only added a minor suggestion, but I think it can be merged!!
Good job @uristdwarf :+1:

**Pull request merged**